database/migrations/create_verb_snapshots_table.php

```diff
@@ -9,7 +9,11 @@
     public function up()
     {
         Schema::create('verb_snapshots', function (Blueprint $table) {
-            $table->snowflakeId();
+            match (config('verbs.id_type', 'snowflake')) {
+                'snowflake' => $table->snowflakeId(),
```

Comment:
Should this be the default branch? If not, we may want to throw an exception if it's not set to something we expect.

```suggestion
                default => $table->snowflakeId(),
```